backend/apps/accounts/views.py

```
@@ -515,509 +120,46 @@ def post(self, request):
-    permission_classes = [permissions.AllowAny]
-    throttle_classes = [MagicLinkVerifyThrottle]
-
-    def post(self, request):
-        serializer = MagicLinkVerifySerializer(data=request.data)
-        serializer.is_valid(raise_exception=True)
-
-        token_value = serializer.validated_data["token"]  # type: ignore
-
         try:
-            magic_token = MagicLinkToken.objects.select_related("user").get(
+            reset_token = PasswordResetToken.objects.get(
```

Ruff (0.15.20)

[error] Undefined name `PasswordResetToken`

(F821)